src/NServiceBus.Core/Logging/ColoredConsoleLogger.cs

```diff
@@ -1,26 +1,23 @@
 ﻿namespace NServiceBus
 {
     using System;
-    using System.Runtime.InteropServices;
+    using System.IO;
     using Logging;
 
     static class ColoredConsoleLogger
     {
         static ColoredConsoleLogger()
         {
-            const int STD_OUTPUT_HANDLE = -11;
-            logToConsole = GetStdHandle(STD_OUTPUT_HANDLE) != IntPtr.Zero;
+            logToConsole = Console.OpenStandardOutput() != Stream.Null;
```

Comment:
None of the places I've seen this API used call `Dispose` on the stream, and I see [this](https://referencesource.microsoft.com/#mscorlib/system/io/__consolestream.cs,95) comment in the reference source. Looking at that implementation, calling Dispose doesn't really seem to do anything important.

I'm going to go find the .NET Core implementation to compare, since it will have platform-specific implementations here.